testing/test_uncertain_conditional.py

```diff
@@ -0,0 +1,103 @@
+import gpflow
+import numpy as np
+import unittest
+
+from gpflow.test_util import GPflowTestCase
+from gpflow import settings
+from gpflow.conditionals import conditional, uncertain_conditional
+
+float_type = settings.dtypes.float_type
+
+np.random.seed(0)
+
+class MomentMatchingSVGP(gpflow.models.SVGP):
```

Comment:
Would be great to have some independent equivalence tests between `uncertain_conditional` and the default `conditional` for the case of variance going to zero - independent of external models.

Reply:
See https://github.com/GPflow/GPflow/blob/master/testing/test_conditionals.py and https://github.com/GPflow/GPflow/blob/master/testing/test_kldiv.py for similar tests.